apps/gateway-ui/src/components/WithdrawCard.tsx

```diff
@@ -208,7 +208,11 @@ const ConfirmWithdrawModal = (
             <ModalHeader>{t('withdraw-card.confirm-withdraw')}</ModalHeader>
             <ModalCloseButton />
             <ModalBody>
-              <VStack alignItems='flex-start' justifyContent='space-between'>
+              <Flex
+                direction='column'
+                alignItems='flex-start'
+                justifyContent='space-between'
+              >
```

Comment:
This now needs a gap of `2`, since `VStack` was providing spacing
```suggestion
              <Flex
                direction='column'
                alignItems='flex-start'
                justifyContent='space-between'
                gap={2}
              >
```

Reply:
Makes sense!